app/views/submissions/browse.html.erb

```diff
@@ -8,7 +8,6 @@
           show_sections: <%= Section.exists? %>,
           is_admin: <%= @current_user.admin? %>,
           can_run_tests: <%= @current_user.admin? && @assignment.enable_test %>,
-          max_mark: <%= @assignment.max_mark %>,
```

Comment:
I get what you're doing here, but I wonder if there's a better way here. For a given TA and assignment, the "`max_mark`" is the same for all submissions (depends only on assigned criteria), so it's still possible to pass in a single value here as a prop instead of of including it in every row. I don't feel too strongly about this, but maybe something you could think about for a bit?

Reply:
@mishaschwartz I merged in the PR, but can you please make a separate PR to address this comment? I think you missed it the last round.

Reply:
@david-yz-liu  Yes I thought we agreed that this would be a separate PR